Jenkinsfile

```diff
@@ -1,7 +1,18 @@
-node("maven") {
-  checkout scm
-  stage("Build") {
-    sh "mvn fabric8:deploy -Popenshift -DskipTests"
-  }
-  stage("Deploy")
-}
+node('maven') {
+
+   stage ('Checkout') {
+        checkout scm
+    }
+   stage ('Standalone Integration Tests') {
+        sh "mvn clean verify"
+   }
+
+   stage ('OpenShift Integration Tests') {
+        sh "mvn clean verify -Popenshift-it,openshift,!standalone-it"
```

Comment:
We are not sure about this at the moment because running ITs as part of the pipeline modifies the "production" environment of the user - which should not be done like that. Ideally, it would deploy and run the IT in a separated projects, but we hit OSO limitation.